setup.py

```diff
@@ -26,7 +26,7 @@
     packages         = ['simhash'],
     package_dir      = {'simhash': 'simhash'},
     cmdclass         = {'build_ext': build_ext},
-    dependencies     = [],
+    depends          = [],
```

Comment:
I think you actually want [`install_requires`](https://github.com/seomoz/aardwolf/blob/master/setup.py#L35)